README.md

```diff
@@ -41,6 +41,10 @@ Optional: requires [Handling Amazon Requests manually](#handling-amazon-requests
 
 `npm install hapi --save`
 
+Testing: We use Mocah, Chai, with Expect.js
+
+`npm install mocha chai expect.js --save-dev`
+
```

Comment:
Testing is already mentioned in the end of the README and I would not put it here.
Revert this chunk and see below for suggestions.